### PR TITLE
Update change-address-json-contract.md

### PR DESCRIPTION
**Pull request by jordan-keenan**
Updated Error Codes in change address JSon contract per Tam and Tiffany's request

### CODE REVIEW

**Review by jucarson — COMMENTED** [review]
Looks good Jordan but you may want to consider making the 3 sentences more concise, for example: "502: "Ineligible Account" extended logging error message details:" or something similar.

**Review by rrobison44 — COMMENTED** [review]
Just an idea but since we are using tables for the error codes now, maybe also use a secondary table for the expanded error messages.  For example:
image

File: change-address/change-address-json-contract.md [hunk 1]
```diff
@@ -24,20 +24,50 @@
...
-			 - Occurs when the city, state and zip code fields combined exceeds 38 characters
-		 - "Attempt to enter PO Box as street addr."
- - 504: "Successful Address Update - Name LOC:xxxx"
+| Error Code | Logging Error Message                                                                 |
+| ---------- | ------------------------------------------------------------------------------------- |
+| 500        | "Program running in memo mode" (The program was attempted while the system was in Memo Post Mode) |
+| 501        | [No generic error logging message] (There was an error while attempting to read/process the configuration letter file.) *See notes for logging error message details for each specific configuration file error |
+| 502        | "Ineligible Account" (The member's account type was listed as ineligible in the configuration Letter file) *See notes on possible extended logging error message details for this error code |
+| 503        | "Error processing requested changes:" *See notes on multiple possible extended logging error message details for this error code |
+| 504        | "Successful Address Update - Name LOC:xxxx" |
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply change-address/change-address-json-contract.md]: Done

File: change-address/change-address-json-contract.md [hunk 2]
```diff
@@ -24,20 +24,50 @@
...
 
+*Expanded Error Code Notes
+- 501: “Ineligible Account”, it can have logging error message details for each of these specific configuration file errors:
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply change-address/change-address-json-contract.md]: Done

**Review by jucarson — APPROVED** [review]
Looks Good Jordan. Thanks!

**Review by rrobison44 — APPROVED** [review]
Updates look good.

**Review by tkainz — APPROVED** [review]
Thanks Jordan

File: change-address/change-address-json-contract.md [hunk 3]
```diff
@@ -24,20 +24,50 @@
...
+	- "CFG open error "[with the config file name]":"[with system generated letter file open error message]
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply change-address/change-address-json-contract.md]: done

File: change-address/change-address-json-contract.md [hunk 4]
```diff
@@ -24,20 +24,50 @@
...
+	- "Invalid Clear Warning Type Setup"
+	- "Invalid Set Warning Type Setup"
+	- "PO Box Types Setup Incomplete"
+	- "No name types set for name level matching"
+	- "Invalid test account found. Must be 10 digits long."
+	- "No email detail lines specified for member email."
+	- "Invalid 'from' email address specified for member email."
+	- "Invalid email address specified for Conversations override."
+	
+ - 502: "Ineligible Account", it can also have the following extended logging error message details:
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply change-address/change-address-json-contract.md]: done

**Pull request merged**